src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.split(';');
+
+  const resultingObject = {};
+
+  declarations.forEach((declaration) => {
+    if (declaration.trim() !== '') {
+      const [key, value] = declaration.split(':').map((item) => item.trim());
```

Comment:
It's good that you're trimming the key and value, but you should also consider handling cases where there may be multiple colons in the declaration. For example, a pseudo-element with content: ':' would currently break your code.